packages/csv-parser-price/readme.md

```diff
@@ -0,0 +1,77 @@
+[![commercetools logo][commercetools-icon]][commercetools]
```

Comment:
We can do like other packages and link the page in the official docs for this insteand...